weave/trace_server/refs_internal.py

```diff
@@ -143,9 +144,23 @@ def uri(self) -> str:
         return u
 
 
+@dataclasses.dataclass(frozen=True)
+class InternalArtifactRef:
+    project_id: str
+    id: str
+
+    def __post_init__(self) -> None:
+        # not validating no slashes in project_id because we aren't converting to internal project_id
+        validate_no_slashes(self.id, "id")
+
+    def uri(self) -> str:
+        u = f"{ARTIFACT_REF_SCHEME}:///{self.project_id}/{self.id}"
+        return u
+
+
 def parse_internal_uri(
     uri: str,
-) -> Union[InternalObjectRef, InternalTableRef, InternalCallRef]:
+) -> Union[InternalObjectRef, InternalTableRef, InternalCallRef, InternalArtifactRef]:
```

Comment:
Feels big enough that we can pull out into a type.  It will be easier to reuse in the validation util below

Reply:
will do